lib/lz4.c

```diff
@@ -446,9 +447,9 @@ static const U32 LZ4_skipTrigger = 6;  /* Increase this value ==> compression ru
 *  Local Structures and types
 **************************************/
 typedef enum { notLimited = 0, limitedOutput = 1 } limitedOutput_directive;
-typedef enum { byPtr, byU32, byU16 } tableType_t;
+typedef enum { unusedTable = 0, byPtr = 1, byU32 = 2, byU16 = 3 } tableType_t;
```

Comment:
minor detail : 
My understanding is that `unusedTable` is actually used to describe a table which content has been reset to `0`. I'm wondering if `unusedTable` is best wording to convey this message.

I believe you want to track when a table is in 0-state because in such case it can be converted into anything. In contrast, if it's already filled with something, special care is needed to ensure next usage interprets data correctly.

So, maybe `clearedTable` ?